WordPress/wordpress-playground
# Issue templates/playground online

## Requirements
[Enable issue templates on GitHub](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser)

## What is this PR doing?
Introduces an issue template that is fairly complete for playground.wordpress.net
AKA "Playground online" 

## What problem is it solving?
Provides guidance when creating a new issue for playground.wordpress.net

## How is the problem addressed?
Created an issue template

## Testing instructions
- Check out the [issue template](https://github.com/WordPress/wordpress-playground/pull/1263/files)
